packages/p2p/src/pages/buy-sell/buy-sell-form.jsx

```diff
@@ -105,19 +106,7 @@ const BuySellForm = props => {
         const receive_amount = input_amount * calculated_rate;
         buy_sell_store.setReceiveAmount(receive_amount);
         // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [input_amount, effective_rate]);
-
-    React.useEffect(() => {
-        if (exchange_rate !== previous_rate && rate_type === ad_type.FLOAT) {
-            setPreviousRate(exchange_rate);
-            showModal({
-                key: 'RateChangeModal',
-                props: {
-                    currency: buy_sell_store.local_currency,
-                },
-            });
-        }
-    }, [exchange_rate, previous_rate]);
```

Comment:
No need to show the modal here as we need to show it if there is an error from BE side whilst calling the endpoint. Moved it to the store function below